### DISCUSSION TITLE
Wine bug report regarding the installer

### DISCUSSION THREAD

**Discussion by SraamaR** [mizztgc/infinitas-for-linux#5] [General]
Hi, I filed a Wine bug regarding the installer not working : https://bugs.winehq.org/show_bug.cgi?id=57222

Feel free to help !

**Comment by mizztgc** [comment]
Oh wow, someone already found a fix for the Konaste installers? That's actually pretty impressive, except the installer doesn't allow for quiet/passive installations of the game.
![image](https://github.com/user-attachments/assets/7ee6d422-9a70-4ace-b355-2996d09751d0)

But regardless, this is an absolute game changer in my opinion.
